interface/new/new.php

```diff
@@ -5,7 +5,13 @@
   require("new_comprehensive.php");
```

Comment:
We are always going to be using this. so if the code just pointed to this program instead of new.php that would be great.